Description: remove deps in rebar.test.config
 Without this patch, rebar would download and compile the listed deps
 at compile time.
Author: Philipp Huebner <debalance@debian.org>

Index: erlang-p1-oauth2/rebar.test.config
===================================================================
--- erlang-p1-oauth2.orig/rebar.test.config
+++ erlang-p1-oauth2/rebar.test.config
@@ -52,7 +52,3 @@
 {clean_files,         [".eunit", "ebin/*.beam", "test/*.beam"]}.
 
 
-{deps,
-  [ {meck,   ".*", {git, "https://github.com/eproxus/meck.git",   {tag, "0.8.3"}}}
-  , {proper, ".*", {git, "https://github.com/manopapad/proper.git", {tag, "20e62bc32f9bd43fe2ff52944a4ef99eb71d1399"}}}
-  ] }.
